pkgs/tools/security/iaito/default.nix

```diff
@@ -48,14 +56,15 @@ stdenv.mkDerivation rec {
   };
 
   meta = with lib; {
-    description = "Official frontend of radare2";
+    description = "An official graphical interface of radare2";
     longDescription = ''
-      The official graphical interface for radare2, a libre reverse engineering
-      framework.
+      iaito is the official graphical interface of radare2. It's the
+      continuation of Cutter for radare2 after the Rizin fork.
     '';
-    homepage = "https://github.com/radareorg/iaito";
+    homepage = "https://rada.re/n/iaito.html";
     changelog = "https://github.com/radareorg/iaito/releases/tag/${src.rev}";
     license = licenses.gpl3Plus;
     maintainers = with maintainers; [ azahi ];
+    platforms = platforms.linux;
   };
```

Comment:
Could these changes be broken up into separate commits? (Maybe something like `iaito: update meta` or `iaito: update description and homepage` and `iaito: add platforms` or similar?)

Also: why the description change so soon? Originally I thought this was leftover from the r2 Cutter days but this package is new. Is this new one from upstream or something? Ditto on the homepage -- was that not a thing when you made the package originally, or did you just not know that it existed?

Reply:
Its alright to do this in one commit. If it would have beeen done in multiple it might even be squash merged.

Reply:
I'll leave it as-is.